src/melee.cpp

```diff
@@ -1194,11 +1194,12 @@ void player::perform_technique(ma_technique technique, Creature &t, int &bash_da
     }
 
     if (technique.knockback_dist > 0) {
-        int kb_offset = rng(
-                            -technique.knockback_spread,
-                            technique.knockback_spread
-                        );
-        t.knock_back_from(posx() + kb_offset, posy() + kb_offset);
+        const int kb_offset_x = rng( -technique.knockback_spread,
```

Comment:
Super nit-pick, this should ideally be a random distance and a random rotation based on rl_dist(), but I doubt the distances are high enough to make a difference in practice.